src/features/query-bar/components/filter/ValueField.js

```diff
@@ -70,20 +73,32 @@ const ValueField = ({ onChange, selectedValue, valueOptions, valueType }: Props)
                         hideLabel
                         label="Date"
                         name="datepicker"
-                        onChange={e => onChange(e.toString())}
+                        onChange={e => {
+                            return e ? onChange([e.toString()]) : onChange([]);
```

Comment:
Could it fire with an empty event? I'm wonder if this ternary op is needed?

Reply:
It still fires with a null event.